Detectors/GlobalTracking/src/MatchHMP.cxx

```diff
@@ -377,14 +398,20 @@ void MatchHMP::doMatching()
       auto& trefTrk = trackWork.first;
 
       prop->getFieldXYZ(trefTrk.getXYZGlo(), bxyz);
-      Double_t bz = -bxyz[2];
+      double bz = -bxyz[2];
 
-      double timeUncert = 100.; //= trackWork.second.getTimeStampError();
+      double timeUncert = trackWork.second.getTimeStampError();
 
-      float minTrkTime = (trackWork.second.getTimeStamp() - mSigmaTimeCut * timeUncert) * 1.E3; // minimum track time in ns
-      float maxTrkTime = (trackWork.second.getTimeStamp() + mSigmaTimeCut * timeUncert) * 1.E3; // maximum track time in ns
+      // float minTrkTime = (trackWork.second.getTimeStamp() - mSigmaTimeCut * timeUncert) * 1.E3; // minimum track time in ns
+      // float maxTrkTime = (trackWork.second.getTimeStamp() + mSigmaTimeCut * timeUncert) * 1.E3; // maximum track time in ns
 
-      if (evtTime < (maxTrkTime + timeFromTF) && evtTime > (minTrkTime + timeFromTF)) {
+      float minTrkTime = (trackWork.second.getTimeStamp() - mSigmaTimeCut * timeUncert);
+      float maxTrkTime = (trackWork.second.getTimeStamp() + mSigmaTimeCut * timeUncert);
+
+      float deltaTime = evtTime - trackWork.second.getTimeStamp();
+
+      // if (evtTime < (maxTrkTime + timeFromTF) && evtTime > (minTrkTime + timeFromTF)) {
+      if (evtTime < maxTrkTime && evtTime > minTrkTime) {
```

Comment:
deltaTime is not used, either use it as 
```
float deltaTime = evtTime - trackWork.second.getTimeStamp();
if (std::abs(deltaTime) < mSigmaTimeCut * timeUncert) { 
...
```
and suppress minTrkTime and maxTrkTime or vice versa.

Reply:
deltaTime was for testing, then I forgot to cancel!